tests/test_fr.py

```diff
@@ -156,6 +156,29 @@
     </select>
 '''
 
+NEW_REG_SELECT_OUTPUT = '''
+    <select name="reg">
+        <option value="01">01 - Guadeloupe</option>
+        <option value="02">02 - Martinique</option>
+        <option value="03">03 - Guyane</option>
+        <option value="04">04 - La Réunion</option>
+        <option value="06">06 - Mayotte</option>
+        <option value="11">11 - Île-de-France</option>
+        <option value="24">24 - Centre-Val de Loire</option>
+        <option value="27">27 - Bourgogne-Franche-Comté</option>
+        <option value="28">28 - Normandie</option>
+        <option value="32">32 - Nord-Pas-de-Calais-Picardie</option>
+        <option value="44">44 - Alsace-Champagne-Ardenne-Lorraine</option>
+        <option value="52" selected="selected">52 - Pays de la Loire</option>
```

Comment:
Why is this a particular region preselected by default? This one isn't a bad choice, but wouldn't it be best not to select any?